src/node/internal/web_crypto.d.ts

```diff
@@ -1,12 +1,4 @@
 // Just enough of web crypto api to write the code in node-compat
-
-declare namespace crypto {
-  function getRandomValues<T extends ArrayBufferView | null>(array: T): T;
-  function randomUUID(): string;
-
-  const subtle: SubtleCrypto;
-}
```

Comment:
This change doesn't seem to relate to the rest?

Reply:
updating `@types/node` makes this code redundant, and actually fails the linter, because of duplication. (latest `@types/node` already has the same exact namespace and function definitions.)

Reply:
Aha!